jobs/config.json

```diff
@@ -3012,6 +3012,22 @@
       "UNKNOWN"
     ]
   },
+  "sig-cli-ci-kubernetes-e2e-gce-serial": {
```

Comment:
The name probably should be `ci-kubernetes-e2e-gce-sig-cli-serial` or `ci-kubernetes-e2e-gce-serial-sig-cli` to be consistent with other jobs :-)